lib/cli.js

```diff
@@ -96,27 +87,33 @@ function runYo(env) {
   router.navigate('home');
 }
 
-rootCheck('\n' + chalk.red('Easy with the `sudo`. Yeoman is the master around here.') + '\n\nSince yo is a user command, there is no need to execute it with root\npermissions. If you\'re having permission errors when using yo without sudo,\nplease spend a few minutes learning more about how your system should work\nand make any necessary repairs.\n\nA quick solution would be to change where npm stores global packages by\nputting ~/npm/bin in your PATH and running:\n' + chalk.blue('npm config set prefix ~/npm') + '\n\nSee: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md');
+require('root-check')([
+  '',
+  chalk.red('Easy with the `sudo`. Yeoman is the master around here.'),
+  '',
+  'Since ' + chalk.green('yo') + ' is a user command, there is no need to execute it with root permissions. If you have permission errors when using yo without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.',
+  '',
+  'A quick solution would be to change where npm stores global packages by putting ~/npm/bin in your PATH and running:',
+  chalk.blue('npm config set prefix ~/npm'),
+  '',
+  'See: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md'
+].join('\n'));
 
 var insightMsg = chalk.gray('==========================================================================') +
-chalk.yellow('\nWe\'re constantly looking for ways to make ') + chalk.bold.red(pkg.name) +
+chalk.yellow('\nWe are constantly looking for ways to make ') + chalk.bold.red(pkg.name) +
 chalk.yellow(
   ' better! \nMay we anonymously report usage statistics to improve the tool over time? \n' +
   'More info: https://github.com/yeoman/insight & http://yeoman.io'
 ) +
 chalk.gray('\n==========================================================================');
 
-var insight = new Insight({
+var insight = new (require('insight'))({
   trackingCode: 'UA-31537568-1',
   packageName: pkg.name,
   packageVersion: pkg.version
 });
 
-if (opts.insight === false) {
-  insight.config.set('optOut', true);
-} else if (opts.insight) {
-  insight.config.set('optOut', false);
-}
+insight.config.set('optOut', opts.insight === false);
```

Comment:
you're right. I completely glossed over the else if part. But what else would `opts.insight` be, `undefined || 0`?